steering_vectors/train_steering_vector.py

```diff
@@ -27,7 +27,7 @@ def train_steering_vector(
     layer_type: LayerType = "decoder_block",
     layer_config: Optional[ModelLayerConfig] = None,
     move_to_cpu: bool = False,
-    read_token_index: int = -1,
+    read_token_index: int | List[int] | Tensor = -1,
```

Comment:
nit: here as well, just use `list` since we're using 3.10+ (we should add a linting rule for this...)

Reply:
Done in [83b0ccf](https://github.com/steering-vectors/steering-vectors/pull/21/commits/83b0ccff5fa9fdedafe08830592102e55fd0b980)